WooCommerce/WooCommerce.xcodeproj/xcshareddata/xcschemes/WooCommerce.xcscheme

```diff
@@ -123,6 +123,10 @@
             argument = "mocked-wpcom-api"
             isEnabled = "NO">
          </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-force-crash-logging 1"
```

Comment:
This [is read in the Tracks library](https://github.com/Automattic/Automattic-Tracks-iOS/blob/b030af97b980fc4c1c0ac8fc66c2d9c3dff770b8/Automattic-Tracks-iOS/Crash%20Logging/CrashLogging.swift#L88) – it allows overriding the "disable crash logging in debug builds" setting if you want to test PRs like this one in debug builds. The downside is that you don't get symbolication, but it can be handy to have.

Reply:
Ahh. Thank you!